pkg/injection/namespace/mapper/mapper.go

```diff
@@ -110,7 +110,7 @@ func matchMetadataEnrichment(dk *dynatracev1beta2.DynaKube, namespace *corev1.Na
 // updateNamespace tries to match the namespace to every dynakube with codeModules
 // finds conflicting dynakubes(2 dynakube with codeModules on the same namespace)
 // adds/updates/removes labels from the namespace.
-func updateNamespace(namespace *corev1.Namespace, deployedDynakubes *dynatracev1beta2.DynaKubeList) (bool, error) {
+func updateNamespace(namespace *corev1.Namespace, deployedDynakubes *dynakube.DynaKubeList) (bool, error) {
```

Comment:
```suggestion
func updateNamespace(namespace *corev1.Namespace, deployedDks *dynakube.DynaKubeList) (bool, error) {
```
Not quite sure about this one, though.

Reply:
let's not touch this one here, we can fix it separately. It doesn't affect directly imports.